<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convert number</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-image: url('C:/qtCreator/labs/lisaOp1/image.jpg');
   background-position: center;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	padding-top:3px;
	background-color:pink;
	border-radius: 5px;
}
QTextEdit:hover{
	opacity:0.4;
	border: 1px solid black
}
</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="placeholderText">
     <string>Insert number</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxIn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 0px;
}

QRadioButton{
	padding-left:20px;
}

QRadioButton::indicator {
                            width: 11px;
                            height: 11px;
                            border: 1px solid white;
                            border-radius:6px;
}

QRadioButton::indicator:checked {
	background-color: pink;
}

QRadioButton::indicator:unchecked {
	background-color: white;
}</string>
    </property>
    <property name="title">
     <string>From:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="radioButton10In">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton2In">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton8In">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonXIn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton::indicator {
    width: 0;
    height: 0;
    border: none;
}

QRadioButton{
	color: transparent;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxOut">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 0px;
}

QRadioButton{
	padding-left:12px;
}

QRadioButton::indicator {
                            width: 11px;
                            height: 11px;
                            border: 1px solid white;
                            border-radius:6px;
}

QRadioButton::indicator:checked {
	background-color: pink;
}

QRadioButton::indicator:unchecked {
	background-color: white;
}</string>
    </property>
    <property name="title">
     <string>To:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="radioButton10Out">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
      </font>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton2Out">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton8Out">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonXOut">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton::indicator {
    width: 0;
    height: 0;
    border: none;
}

QRadioButton{
	color: transparent;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border-radius: 5px;
	background-color: pink;
	padding-left:10px;
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 5px;
	background-color: pink;
	color: gray;
}
</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copyButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 5px;
	background-color: pink;
}

QPushButton:hover{
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newCCButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 5px;
	background-color: pink;
}

QPushButton:hover{
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>New system</string>
    </property>
   </widget>
   <widget class="QPushButton" name="swapButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 5px;
	background-color: pink;
}

QPushButton:hover{
	border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Swap</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
